### DISCUSSION TITLE
Support for Macedonian?

### DISCUSSION THREAD

**Discussion by vvitanov** [typst/typst#5601] [Q&A] [answered]
Does Typst support Macedonian? (hyphenation, translated labels, etc.)

**Comment by laurmaedje** [comment, marked as answer]
Not currently. 
- Hyphenation is not supported because the relevant hyphenation patterns [are GPL-licensed](https://github.com/hyphenation/tex-hyphen/blob/5ae744dbee402f9ef148c59d8479883925c7b59c/hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-mk.tex#L6). 
- Translations are not available, but you could contribute them. You can [search through previous PRs](https://github.com/typst/typst/pulls?q=is%3Apr+translations+is%3Aclosed) to see how.
